dotnet/src/Connectors/Connectors.AzureOpenAI/AzureOpenAIChatCompletionExecutionSettings.cs

```diff
@@ -17,7 +17,7 @@ namespace Microsoft.SemanticKernel.Connectors.AzureOpenAI;
 /// Execution settings for an AzureOpenAI completion request.
 /// </summary>
 [JsonNumberHandling(JsonNumberHandling.AllowReadingFromString)]
-public sealed class AzureOpenAIPromptExecutionSettings : PromptExecutionSettings
+public sealed class AzureOpenAIChatCompletionExecutionSettings : PromptExecutionSettings
```

Comment:
This introduces a new pattern for just Azure OpenAI making it inconsistent without other connectors.

I don't know if we want to rename `OpenAIPromptExecutionSettings` as this will be a very disruptive change. Can we consider this change in a seperate PR.

Reply:
The change has been rolled back.